python/lsst/meas/extensions/ngmix/bootstrap.py

```diff
@@ -144,7 +144,7 @@ def fit_psfs(self):
                 tres['g2'] = g2
                 tres['T'] = T
             else:
-                filt = self.cdict['filters'][band]
+                filt = self.cdict['bands_fit'][band]
```

Comment:
What is the motivation to rename this?

Reply:
This is part of the interface for the `MultibandFit` pipelinetask I wrote for MultiProFit: https://github.com/lsst/pipe_tasks/blob/master/python/lsst/pipe/tasks/fit_multiband.py

As for why it's `bands_fit`, `bands` vs `filters` is part of a naming scheme change throughout the stack where `filter` is reserved for physical filter glass, while `band` is used for, well, bands. In fact you can't build this package with recent releases anymore because the `getShortFilterName` method from `lsst.pipe.tasks.multiBand` no longer exists. The `_fit` suffix is for specificity, because you can optionally specify additional bands to load data for but not actually fit (e.g. if you want a magnitude-based prior based on an existing measurement in a band that you're not fitting).

Reply:
got it